lib/linguist/languages.yml

```
@@ -6573,6 +6573,9 @@ Unix Assembly:
   extensions:
   - ".s"
   - ".ms"
+  aliases:
+  - at&tasm
+  - unixasm
```

```suggestion
  - gas
  - gnu asm
  - unix asm
```

Multi-word aliases will match dashed variants (`gnu-asm`, `unix-asm`), as well as single-token variants (`gnuasm`, `unixasm`), so it's always better to specify aliases as separate words when possible.

I also recommend dropping `at&asm`: it's not very obvious as an abbreviation of _"Unix Assembly"_.